cairo/ethereum/cancun/vm/gas.cairo

```diff
@@ -300,9 +300,14 @@ func calculate_data_fee{range_check_ptr}(excess_blob_gas: U64, tx: Transaction)
     alloc_locals;
     // saturate at 2**64 - 1
     let total_blob_gas = calculate_total_blob_gas(tx);
-    // saturate at 2**64 - 1
+    // saturate at 2**128 - 1
```

Comment:
Should calculate_total_blob_gas be saturate also at 2**128 - 1 ?

Reply:
Not necessarily. This is kind of overfitting for tests, 2**64 would've been fine as well as the TX would've been killed in a later part of the tx

Reply:
ok fine for me